src/include/pg_version_compat.h

```diff
@@ -93,6 +108,17 @@ getAttstattarget_compat(HeapTuple attTuple)
 }
 
 
+#include "catalog/pg_statistic_ext.h"
+static inline int32
```

Comment:
return `int ` instead for all versions of this compat function. Postgres seems to use `int` for DatumGetInt16 values.